@xen-orchestra/web/src/components/hosts/panel/card/HostNetworkCard.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <UiCard class="card-container">
+    <UiCardTitle>
+      {{ t('network') }}
+    </UiCardTitle>
+    <div class="content">
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('management-ip') }}</template>
+        <template v-if="host.address" #value>{{ host.address }}</template>
+        <template v-if="host.address" #addons>
+          <VtsCopyButton :value="host.address" />
+        </template>
+      </VtsCardRowKeyValue>
+      <template v-if="ipV4Addresses.length > 0">
+        <VtsCardRowKeyValue v-for="(ip, index) in ipV4Addresses" :key="ip">
+          <template #key>
+            <div v-if="index === 0">{{ t('ipv4', ipV4Addresses.length) }}</div>
+          </template>
+          <template #value>{{ ip }}</template>
+          <template #addons>
+            <VtsCopyButton :value="ip" />
+            <UiButtonIcon
+              v-if="index === 0 && ipV4Addresses.length > 1"
+              v-tooltip="t('coming-soon')"
+              disabled
+              icon="fa:ellipsis"
+              size="small"
```

Comment:
```suggestion
              size="medium"
```

<img width="99" height="234" alt="image" src="https://github.com/user-attachments/assets/23da1626-ec85-4acf-a5e2-4b7e4c05a884" />

With small icon it's not aligned ..